housekeeper/store/filters/file_filters.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+
+from housekeeper.store.models import File
+
+
+def filter_files_by_id(files: Query, file_id: int, **kwargs) -> Query:
+    """Filter files by file id."""
+    return files.filter(File.id == file_id)
+
+
+class FileFilters(Enum):
+    """Define File filter functions."""
+    FILTER_BY_ID: Callable = filter_files_by_id
+
+
+def apply_file_filters(
```

Comment:
This function is in plural and the `bundle` and `version` functions in singular. Better change this to singular
```suggestion
def apply_file_filter(
```